shared/teams/team/invite-row/index.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import * as React from 'react'
+import {Avatar, Box, ClickableBox, Text, Usernames} from '../../../common-adapters'
+import {globalStyles, globalMargins} from '../../../styles'
+import {isMobile} from '../../../constants/platform'
+
+export type Props = {
+  username: string,
+  following: boolean,
+  teamname: string,
+  you: ?string,
+  type: ?string,
+}
+
+type TypeMap = {
+  admin: string | boolean,
+  owner: string | boolean,
+  reader: string | boolean,
+  writer: string | boolean,
+}
+
+const typeToLabel: TypeMap = {
```

Comment:
likely should be in const/teams, we might have this in a couple of places already